spec/unit/hanami/router/recognize_spec.rb

```diff
@@ -210,6 +227,21 @@
         expect(route.params).to eq({})
       end
 
+      it 'recognizes redirect' do
```

Comment:
Oh thanks for the suggestion, I wanted to have `#redirect?` as private method to make `RecognizedRoute#action` to return `nil` in case of redirect. But yeah that can make a great way to inspect the route. Let's add it! 😄